spec/classes/datadog_agent_spec.rb

```diff
@@ -1592,17 +1592,37 @@
 
         config_dir = WINDOWS_OS.include?(operatingsystem) ? 'C:/ProgramData/Datadog' : '/etc/datadog-agent'
         config_yaml_file = config_dir + '/datadog.yaml'
+        install_info_file = config_dir + '/install_info'
         log_file = WINDOWS_OS.include?(operatingsystem) ? 'C:/ProgramData/Datadog/logs/agent.log' : '\/var\/log\/datadog\/agent.log'
 
         it { is_expected.to contain_file(config_dir) }
         it { is_expected.to contain_file(config_yaml_file) }
+        it { is_expected.to contain_file(install_info_file) }
         it { is_expected.to contain_file(config_dir + '/conf.d').with_ensure('directory') }
 
         # Agent 5 files
         it { is_expected.not_to contain_file('/etc/dd-agent') }
         it { is_expected.not_to contain_concat('/etc/dd-agent/datadog.conf') }
         it { is_expected.not_to contain_file('/etc/dd-agent/conf.d').with_ensure('directory') }
 
+        describe 'install_info check' do
+          it {
+            is_expected.to contain_file(install_info_file).with(
```

Comment:
Why not parse it as YAML in Ruby and check the keys' values are correct?

Reply:
Sure! I was trying to follow the style of the existing tests, I guess we should refactor those as well at some point 🤔